options/locale/locale_en-US.ini

```diff
@@ -155,6 +155,30 @@ filter.not_template = Not Template
 filter.public = Public
 filter.private = Private
 
+[search]
+search = Search
+type_tooltip = Search type
+fuzzy = Fuzzy
+fuzzy_tooltip = Include results that also match the search term closely
+match = Match
+match_tooltip = Include only results that match the exact search term
+repo_kind = Search repos...
+repo_no_results = No matching repositories found.
+user_kind = Search users...
+user_no_results = No matching users found.
+org_kind = Search orgs...
+org_no_results = No matching organizations found.
+team_kind = Search teams...
+code_kind = Search code...
+code_no_results = No matching source code found.
+code_search_unavailable = Code search is currently not available. Please contact your site administrator.
```

Comment:
For a large instance, "your" seems not a proper word.

For example, if GitHub crashes, GitHub users couldn't find "their" site administrator. 🤣 


So maybe it could be "Please contact the site administrator."